### DISCUSSION TITLE
RuntimeError: CUDA error: device-side assert triggered

### DISCUSSION THREAD

**Discussion by Karthikeyan-Ram** [pyg-team/pytorch_geometric#3585] [Q&A]
while running the following code in colab pro (either GPU or TPU), getting the error. 
"RuntimeError: CUDA error: device-side assert triggered"

device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
data = data.to(device)

**Comment by rusty1s** [comment]
Do you have a small script to reproduce?
